transform/mattermost-analytics/models/staging/mm_mobile_prod/_mm_mobile_prod__models.yml

```diff
@@ -0,0 +1,22 @@
+version: 2
+
+models:
+  - name: stg_mm_mobile_prod__tracks
+    description: |
+      Reconstructed `tracks` table using custom properties expected to be in the events.
+
+    columns:
```

Comment:
does it need to contain `server_id` as well?

Reply:
Just pointing out that it's missing in this columns list, if we plan to have it.

Reply:
Ah, sorry, missed that. Thanks, nice catch, added it!